### PR TITLE
Improving FreeMarker feature doc

### PR DESCRIPTION
**Pull request by nilminiwso2**
Improving FreeMarker feature doc

### CODE REVIEW

**Review by Vathsan — APPROVED** [review]

**Comment by nilminiwso2** [comment]
On hold. Will merge this once tooling support is available with the next WSO2 Integration Studio release.

**Comment by nilminiwso2** [comment]
Closing this PR for now. Will apply these changes when the next Integration Studio version is released.

**Pull request closed without merging**
